File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.js
function solution(participant, completion) {
    var answer = '';
    //동명이인 있을 수 있단 게 포인트 
    participant.sort()
    completion.sort()
    for (let i=0;i<participant.length;i++){

        if(participant[i]===completion[i]){
            continue
        }else{
            answer=participant[i]
            break
        }
    }
    
    return answer;
}

File: 프로그래머스/lv1/42576. 완주하지 못한 선수/완주하지 못한 선수.py
def solution(participant, completion):
    answer = ''
    participant.sort()
    completion.sort()
    for i in range(len(participant)):
        if i> len(completion)-1:
            answer=participant[i]
        elif participant[i] !=completion[i]:
            answer=participant[i]
            break

    return answer